collectors/ebpf.plugin/README.md

```diff
@@ -208,3 +229,7 @@ Our [initial testing](https://github.com/netdata/netdata/issues/8195) shows the
 nearly identical to our [apps.plugin collector](/collectors/apps.plugin/README.md), despite collecting and displaying
 much more sophisticated metrics. You can now use the eBPF to gather deeper insights without affecting the performance of
 your complex applications at any load.
+
+## Integration with Apps.plugin 
+
+Beginning with v1.23 of the Agent, `ebpf.plugin` integrates with [`apps.plugin`](/collectors/apps.plugin/README.md) to create charts that help you understand how applications interact with the Linux kernel.
```

Comment:
We can remove this section entirely, since we'll talk about the integration higher in the document.

Reply:
I removed it.